refactor: 💡 structure changes for automation chapter (#101)
virtualidentityag/dev-fe-onboarding: src/automation.md

<img src="./assets/gorilla_badge.svg" width="30%" height="auto" alt="groovy_gorilla">

> Last chapter you have got more insights in our Biotope Ecosystem and now you are able to use it in our projects and help us improving it. That's great! In this chapter you will learn more about automation and automation tools like GitHub Actions.

Reviewer: ```suggestion
Last chapter you have got more insights in our Biotope Ecosystem and now you are able to use it in our projects and help us improving it. That's great! In this chapter you will learn about automation and the automation tool GitHub Actions.
```

> Goal: You know how to implement an automated workflow with GitHub Actions in your projects.

---
## Preparation
Developer and operations are no longer the exclusive responsibility of DevOps engineers alone. Today, Frontend Developers help to reshape the form and the way process occurs within their projects improving collaboration between Frontends, Backends, and Stakeholders themselves. <br/>

> Testing, building, authenticating and deploying software can be done in a more smooth and automatic way by implementing principles of **Continuous Integration and Continuous Delivery** (CI/CD, for short).<br/>

Reviewer: ```suggestion
Testing, building, authenticating and deploying software can be done in a more smooth and automatic way by implementing principles of **Continuous Integration and Continuous Delivery** (CI/CD, for short).

```

> Testing, building, authenticating and deploying software can be done in a more smooth and automatic way by implementing principles of **Continuous Integration and Continuous Delivery** (CI/CD, for short).<br/>

Reviewer: ```suggestion
Testing, building, authenticating and deploying software can be done in a more smooth and automatic way by implementing principles of **Continuous Integration and Continuous Delivery** (CI/CD, for short).

```

        TOKEN: ${{secrets.MY_SUPER_SECRET_STORED_KEY}}
```

Still don't know how this .yml works? click [here](https://www.codeproject.com/Articles/1214409/Learn-YAML-in-five-minutes)

If this section stimulated your curiosity and you want to find out more, click [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-javascript-action)

> ### Questions

Author: Please add Questions for this section.

Reviewer: - What are the benefits from CD/CI?

Would be one I have in mind. Without questions I would not add the Headline. We could start with this one and iteratively add more.

<authors-component v-bind:authors="[
    {

>       username: 'SheepFromHeaven',

Author: @SheepFromHeaven ok? Otherwise add correct responsible person (Tiago is not at VI anymore, right?)
